rest_api/simple_supply_rest_api/transaction_creation.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import hashlib
+
+from sawtooth_rest_api.protobuf import batch_pb2
+from sawtooth_rest_api.protobuf import transaction_pb2
+
+from simple_supply_addressing import addresser
+
+from simple_supply_protobuf import payload_pb2
+
+
+def make_create_agent_transaction(transaction_signer,
+                                  batch_signer,
+                                  name,
+                                  timestamp):
+    """Make a CreateAgentAction transaction and wrap it in a batch
```

Comment:
Is it a `CreateAgentAction` transaction or a `CreateAgent` transaction?

Reply:
Good point, `CreateAgent` would probably make more sense

Reply:
However, the transaction is named CreateAgentAction (same as supplychain 0.8) so if I were to improve this I would do it in another PR.